src/black/strings.py

```diff
@@ -87,7 +87,7 @@ def get_string_prefix(string: str) -> str:
     prefix = ""
     prefix_idx = 0
     while string[prefix_idx] in STRING_PREFIX_CHARS:
-        prefix += string[prefix_idx].lower()
+        prefix += string[prefix_idx]
```

Comment:
I wonder if this could not repeatedly append to the string and just calculate the index, and then slice the whole prefix at once 🤔 Not really a comment to the PR but came to my mind anyway.

Reply:
Sadly `min(s.find("'"), s.find('"'))` doesn't quite work because no matches is returned as -1 😄

Reply:
I'm taking this as that no you have no nits or objections for this PR :)

Thanks for the review though! You're doing great adjusting to your new role!